docs/pt/docs/deployment/manually.md

```diff
@@ -0,0 +1,169 @@
+* <a href="https://github.com/django/daphne" class="external-link" target="_blank">Daphne</a>: servidor ASGI construído para Django Channels.
+* <a href="https://github.com/emmett-framework/granian" class="external-link" target="_blank">Granian</a>: um servidor HTTP Rust para aplicações Python.
+* <a href="https://unit.nginx.org/howto/fastapi/" class="external-link" target="_blank">NGINX Unit</a>: NGINX Unit é um runtime de aplicação web leve e versátil.
+
+## Máquina Servidora e Programa Servidor
+
+Existe um pequeno detalhe sobre estes nomes para se manter em mente. 💡
+
+A palavra "**servidor**" é comumente usada para se referir tanto ao computador remoto/nuvem (a máquina física ou virtual) quanto ao programa que está sendo executado nessa máquina (por exemplo, Uvicorn).
+
+Apenas tenha em mente que quando você ler "servidor" em geral, isso pode se referir a uma dessas duas coisas.
+
+Quando se refere à máquina remota, é comum chamá-la de **servidor**, mas também de **máquina**, **VM** (máquina virtual), **nó**. Todos esses termos se referem a algum tipo de máquina remota, normalmente executando Linux, onde você executa programas.
+
+## Instale o Programa Servidor
+
+Quando você instala o FastAPI, ele vem com um servidor de produção, o Uvicorn, e você pode iniciá-lo com o comando `fastapi run`.
+
+Mas você também pode instalar um servidor ASGI manualmente.
+
+Certifique-se de criar um [ambiente virtual](../virtual-environments.md){.internal-link target=_blank}, ativá-lo e, em seguida, você pode instalar a aplicação do servidor.
+
+Por exemplo, para instalar o Uvicorn:
+
+<div class="termy">
+
+```console
+$ pip install "uvicorn[standard]"
+
+---> 100%
+```
+
+</div>
+
+Um processo semelhante se aplicaria a qualquer outro programa de servidor ASGI.
+
+/// tip | Dica
+
+Adicionando o `standard`, o Uvicorn instalará e usará algumas dependências extras recomendadas.
+
+Isso inclui o `uvloop`, a substituição de alto desempenho para `asyncio`, que fornece um grande aumento de desempenho de concorrência.
+
+Quando você instala o FastAPI com algo como `pip install "fastapi[standard]"`, você já obtém `uvicorn[standard]` também.
+
+///
+
+## Execute o Programa Servidor
+
+Se você instalou um servidor ASGI manualmente, normalmente precisará passar uma string de importação em um formato especial para que ele importe sua aplicação FastAPI:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --host 0.0.0.0 --port 80
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://0.0.0.0:80 (Press CTRL+C to quit)
+```
+
+</div>
+
+/// note | Nota
+
+O comando `uvicorn main:app` refere-se a:
+
+* `main`: o arquivo `main.py` (o "módulo" Python).
+* `app`: o objeto criado dentro de `main.py` com a linha `app = FastAPI()`.
+
+É equivalente a:
+
+```Python
+from main import app
+```
+
+///
+
+Cada programa de servidor ASGI alternativo teria um comando semelhante, você pode ler mais na documentação respectiva.
+
+/// warning | Aviso
```

Comment:
You just missed the quotes:

/// warning | ***"*Aviso**"**

Reply:
Fixed in: https://github.com/fastapi/fastapi/pull/12210/commits/99ce96f62e6f1780b2ed94d7e809966c15c9ae1d